cmd/metrics-v2.go

```diff
@@ -942,94 +942,109 @@ func getMinioProcMetrics() *MetricsGroup {
 			logger.LogOnceIf(ctx, err, nodeMetricNamespace)
```

Comment:
I reproduced the error, and looks like it is getting triggered at this line (942).

Reply:
This line of error is okay and we should print this error at least @anjalshireesh not the other errors.

Reply:
that is why it is left here on purpose